pycodestyle.py

```diff
@@ -131,7 +132,8 @@ def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
     'and', 'in', 'is', 'or'] +
     FUNCTION_RETURN_ANNOTATION_OP +
     ASSIGNMENT_EXPRESSION_OP)
-WHITESPACE = frozenset(' \t')
+NON_BREAKING_WHITESPACE = ' '
```

Comment:
please use the escape sequence instead otherwise this code is very difficult to read

Reply:
Done in https://github.com/PyCQA/pycodestyle/pull/1035/commits/6975080d6b6d7cb647b35f8abdcbea6c986ba0c0